PROTOCOL.md

```diff
@@ -157,9 +157,9 @@ represented by a space), and the rest of the string being their username.
 
 > `USER` left the room.
 
-`|name|USER|OLDID` or `|n|USER|OLDID`
+`|name|USER|OLDID|AWAY` or `|n|USER|OLDID|AWAY`
 
-> A user changed name to `USER`, and their previous userid was `OLDID`.
+> A user changed name to `USER`, and their previous userid was `OLDID`. If a user is marked as away, the away state is appended at the end of the message. This is not present for users without an away state.
```

Comment:
More explanation on what values `AWAY` could take on? I'm assuming its just a literal string `'away'` if it's present?